agent/custom/action/combat.py

```
@@ -883,19 +936,26 @@ def run(
-        elif available_count <= 0:
+        availability = _tc_get_availability(context)
+        available_count = availability.available_count
+        if availability.page is _TargetCountPage.UNKNOWN or available_count is None:
+            logger.error("无法识别关卡体力信息，停止重开关卡")
+            context.run_task("HomeButton")
+            context.tasker.post_stop()
+            return CustomAction.RunResult(success=False)
+
+        if available_count <= 0:
             logger.debug("没体力咯，吃个糖")
             for _ in range(2):  # 最多吃两次糖，防止吃mini糖体力不够
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)